typings.d.ts

```diff
@@ -409,8 +409,9 @@ declare module 'launchdarkly-js-sdk-common' {
 
     /**
      * An object describing the main factor that influenced the flag evaluation value.
+     * Will be `null` if the `evaluationReasons` client option is `false`.
```

Comment:
Let's change this to:

```suggestion
     * Will be `null` if `[[LDOptions.evaluationReasons]]` is `false`.
```